libnavui-tripprogress/src/main/java/com/mapbox/navigation/ui/tripprogress/view/MapboxTripProgressView.kt

```diff
@@ -52,18 +52,34 @@ class MapboxTripProgressView : FrameLayout {
     private val binding =
         MapboxTripProgressLayoutBinding.inflate(
             LayoutInflater.from(context),
-            this,
-            true
+            this
         )
 
+    /**
+     * [TimeRemainingView] to render time remaining to reach the destination
+     */
+    val timeRemainingView: TimeRemainingView = binding.timeRemainingText
+    /**
+     * [TimeRemainingView] to render distance remaining to reach the destination
+     */
+    val distanceRemainingView: DistanceRemainingView = binding.distanceRemainingText
+    /**
+     * [TimeRemainingView] to render estimated arrival time to reach the destination
+     */
+    val estimatedArrivalTimeView: EstimatedArrivalTimeView = binding.estimatedTimeToArriveText
```

Comment:
Do we need to make these references (and classes as well) public? If we're exposing the styling options through XML, we should be able to hide the references and classes which opens up a door for potential future refactors that won't break.

I understand exposing the whole view is more flexible for developers but at a maintenance cost and I think we could start with smaller API surface to be opened if needed.

Reply:
I have removed this and introduced `TripViewOptions`. I like this solution so much better.